### DISCUSSION TITLE
Not installing

### DISCUSSION THREAD

**Discussion by Ghoulforlife** [termux/termux-app#3481] [Q&A] [answered]
i download the app and it wasn't installing i got an android 12 2GB ram i don't know if ram is needed to install it

**Comment by agnostic-apollo** [comment, marked as answer]
2GB ram itself won't be an issue. Read https://github.com/termux/termux-app#Installation or otherwise check adb logcat log for reason for installation failure. There are installer apps that will show reason too, check https://github.com/termux/termux-app/issues/3206#issuecomment-1403768162
